_posts/2018-01-22-LGBTSteminar2018.md

```diff
@@ -0,0 +1,23 @@
+
+We cannot entirely separate our work from our lives, and so it is important that we can be ourselves at work.  So many of the people at this conference lacked LGBT role models when they were growing up, and are finding themselves becoming these role models whether they like it or not.  Whether we like it or not, it is our responsibility to make this path easier for future generations.  This is a big responsibility, and we need to work together in order to do this well.  One of the common themes throughout all of the talks was the importance of being visibly out.  Whilst we cannot and should not ask individuals to come out, the act of being explicitly queer is of immense value to other LGBT+ people who are still discovering themselves.  A small number of people coming out can start to create a critical mass, making others feel safe(r) to be themselves in the workplace.
+
+If your workplace or community lacks queer spaces then make them happen.  If these safe spaces get big then the system will accept it and say it was a wonderful idea all along.  If not, it won't be noticed if you fail.  When you're lonely, this is the best time to try and build a community.
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">. <a href="https://twitter.com/PhdGeek?ref_src=twsrc%5Etfw">@phdgeek</a>&#39;s guide to getting stuff done <a href="https://twitter.com/hashtag/LGBTSTEMinar18?src=hash&amp;ref_src=twsrc%5Etfw">#LGBTSTEMinar18</a> <a href="https://t.co/CZuZ4fVoel">pic.twitter.com/CZuZ4fVoel</a></p>&mdash; Alice From Online (@AliceFromOnline) <a href="https://twitter.com/AliceFromOnline/status/951758942270615552?ref_src=twsrc%5Etfw">January 12, 2018</a></blockquote>
+<script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+There are a large number of existing meetup groups for specific technologies in our city as well as chapters within our own organisations, and whilst these are of tremendous value the environments can be alienating to people who do not share the same experiences as the majority of the group.  An explicitly LGBT+ space provides somewhere for LGBT+ people to be themselves and talk about their experiences without fear of judgement.  Spaces like the LGBT Steminar feel like a very different environment to STEM events with no spatial relationship to sexuality.  With presenters feeling able to be themselves without fear they give much better presentations, and the audience is much more appreciative.  The applause goes on for much longer, there's much more laughter and joking, and the atmosphere is generally much happier.
```

Comment:
> in our city

Do we want to be leeds-specific in this?

Reply:
removed the city-specific wording